apps/are_form/lib/classARE.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
```

Comment:
I don't like classes and I hate `this`

`this` makes bugs

I'd love for this to be more functional but whatever

Reply:
classARE is a strange name